app/src/components/ButtonsOverlay.tsx

```diff
@@ -8,6 +8,7 @@ export const ButtonsOverlay = (): React.ReactElement => {
 
   const onAboutClicked = () => {
     localStorage.removeItem('welcomeComplete');
+    localStorage.removeItem('notificationComplete');
```

Comment:
@arthur-fox this seems wierd - why do we want them to see the overlays again if they go to the about page?

Reply:
It was a quick way to solve the case where people accidentally clicked past it the first time without reading... if they clicked on the About page then they clearly didn't understand how everything works... would definitely be happy with a more intelligent way of doing it though

Reply:
maybe we can just add this information to the about page? that way if they have gone there then they will see it regardless?